### ISSUE TITLE
Booklet Init crashes with PageFurnace

### ISSUE THREAD

**Issue by Thenosis** [Ellpeck/ActuallyAdditions#631]
All I know is that I get this crash report when trying to start my custom modpack. I'm not proficient at reading crash reports yet so I can't narrow down the possible cause. 

http://pastebin.com/auz7R6K0

**Comment by Ellpeck** [comment]
You appear to have removed a Furnace Recipe, possibly using MineTweaker. Is that true?

**Comment by Ellpeck** [comment]
Actually, this is caused by some mod adding a broken Furnace Recipe that doesn't contain an output item.

**Comment by Thenosis** [comment]
Oh! I hadn't even changed any of the recipes yet when the crash occurred. Just thought I'd let you know about it. I think it may have been due to Ex Nihilo Adscensio and adding an AbyssalCraft ore to the sieve table. I think it was something to do with trying to get the Ex Nihilo chunk and dust to smelt into the actual abyssal craft ingot. I've removed Ex Nihilo Adscensio from my modpack as I'm not doing a skyblock anymore. I'll be adding AA back in. Thanks for the help!
